gxcloudstorage-awss3-v1/pom.xml

```diff
@@ -27,7 +27,7 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-java-sdk-s3</artifactId>
-            <version>${com.amazonaws.version}</version>
```

Comment:
why not using the property?

Reply:
I guess the purpose of using replacers is to not repeat the same value over and over again but in this case, this is the only place where this will be used and it also should not be use in other future developments because it is a deprecated SDK